doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -488,16 +488,18 @@ To program the multi-image HEX file, you can use west or nrfjprog.
 
         west flash
 
-   .. group-tab:: nrfjprog
+      .. note::
+          The minimum supported version of nrfjprog for multi-image builds for different cores is 10.21.0.
+
+   .. group-tab:: nRF Util
 
       Enter the following commands to program multi-image builds for different cores::
 
-        nrfjprog -f NRF53 --program zephyr/merged_domains.hex --recover --verify
+        nrfutil device program --firmware zephyr/merged_domains.hex --options verify=VERIFY_READ
+        nrfutil device recover
```

Comment:
So we first program something and then we delete everything?
I suspect that recover do not belong here. Should it be either A: removed, or B: replaced with reset?